cmd/boulder-va/gsb.go

```diff
@@ -64,7 +75,7 @@ func (sb gsbAdapter) IsListed(url string) (string, error) {
 	if err != nil {
 		return "error", err
 	}
-	if len(threats) > 0 {
+	if len(threats) > 0 && threats[0] != nil {
```

Comment:
Anything that needs a numbered explanation comment is definitely worth rewriting :-)

I agree its not very useful to return the `ThreatType` since we're not using it in a meaningful way. I changed it to one `if` and a fixed return in https://github.com/letsencrypt/boulder/pull/2504/commits/e0bf74ecf78b0749bae6f518edce71b1c050526f